hack/hypershift/package-template.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: selectorsyncset-template
+parameters:
+  - name: REGISTRY_IMG
+    required: true
+  - name: IMAGE_DIGEST
+    required: true
+objects:
+  - apiVersion: hive.openshift.io/v1
+    kind: SelectorSyncSet
+    metadata:
+      name: addon-operator-hypershift
+    spec:
+      clusterDeploymentSelector:
+        matchLabels:
+          ext-hypershift.openshift.io/cluster-type: service-cluster
+      resourceApplyMode: Sync
+      resources:
+        - apiVersion: apps.open-cluster-management.io/v1
+          kind: PlacementRule
+          metadata:
+            name: addon-operator
+          spec:
+            clusterConditions:
+              - status: "True"
+                type: ManagedClusterConditionAvailable
+            clusterSelector:
+              matchExpressions:
+                - { key: management-cluster, operator: In, values: [ "true" ] }
+        - apiVersion: policy.open-cluster-management.io/v1
+          kind: PlacementBinding
+          metadata:
+            name: addon-operator
+          placementRef:
+            name: addon-operator
+            kind: PlacementRule
+            apiGroup: apps.open-cluster-management.io
+          subjects:
+            - name: addon-operator
+              kind: Policy
+              apiGroup: policy.open-cluster-management.io
+        - apiVersion: policy.open-cluster-management.io/v1
+          kind: Policy
+          metadata:
+            name: addon-operator
+            namespace: default
```

Comment:
Is this the right namespace in the Service Cluster to create Policy?

Reply:
it is a good start. Policies can be place on the service cluster in any ns. Our product examples are `policies`.

Reply:
I will change to policies then, thanks!

Reply:
does this `policies` namespace exist ?